Frameworks/Foundation/NSFileManager.mm

```diff
@@ -302,18 +302,18 @@ - (BOOL)createDirectoryAtPath:(NSString*)pathAddr
                 bool success = EbrMkdir(curPath);
                 if (!success && errno != EEXIST) {
                     TraceError(TAG, L"Failed to make path %hs: %d", curPath, errno);
-                    // return FALSE;
+                    // return NO;
```

Comment:
there are allot of things that needs to be cleaned up in this file + most of the functionality does not have unit tests. I opened a separate bug for that.
So far this review is for algo's bug fix.

---
In reply to: [71405791](https://github.com/Microsoft/WinObjC/pull/669#discussion_r71405791) [](ancestors = 71405791)